private_gpt/ui/ui.py

```diff
@@ -418,11 +418,54 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
+                    def get_model_label() -> str:
+                        """Get model label from llm mode setting YAML.
+
+                        Raises:
+                            ValueError: If an invalid 'llm_mode' is encountered.
+
+                        Returns:
+                            str: The corresponding model label.
+                        """
+                        # Get model label from llm mode setting YAML
+                        # Labels: local, openai, openailike, sagemaker, mock, ollama
+                        config_settings = settings()
+                        if config_settings is None:
+                            raise ValueError("Settings are not configured.")
+
+                        # Get llm_mode from settings
+                        llm_mode = config_settings.llm.mode
+
+                        # Mapping of 'llm_mode' to corresponding model labels
+                        model_mapping = {
+                            "llamacpp": config_settings.llamacpp.llm_hf_model_file,
+                            "openai": config_settings.openai.model,
+                            "openailike": config_settings.openai.model,
+                            "sagemaker": config_settings.sagemaker.llm_endpoint_name,
+                            "mock": llm_mode,
+                            "ollama": config_settings.ollama.model,
```

Comment:
You need to change this to `config_settings.ollama.llm_model`

Reply:
I updated the config for ollama to `config_settings.ollama.llm_model`: https://github.com/zylon-ai/private-gpt/pull/1648/commits/ab79a93fc860b75f3aeabef15de4a25830707efe